<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Product Registration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Desktop/download-removebg-preview.png</normaloff>../../../../Desktop/download-removebg-preview.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Title">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Product registration</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_products">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>741</width>
      <height>271</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">ID</string>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">Name</string>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">Description</string>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">Price</string>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_CreateProduct">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>420</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Search">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>81</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Search">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>80</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_searchType">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string notr="true">ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">Name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">Description</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">Price</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string notr="true">Category</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_ImportCategories">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Import Categories</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UpdateProduct">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RemoveProduct">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
